web/templates/pages/member_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}TrackCat Member{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="memper_profile">
+				<div class="col-md-12">
+				<h2>{% if user.fullname %}
```

Comment:
@PettraV how about "No!"? :smile: 

now it's a block that's all at the same indentation level, a.k.a. there is no indentation to speak of